angular-color-picker.js

```diff
@@ -1,17 +1,19 @@
 (function (root, factory) {
-    if (typeof define === 'function' && define.amd) {
-        define([ 'module', 'angular' ], function (module, angular) {
-            module.exports = factory(angular);
-        });
-    } else if (typeof module === 'object') {
-        module.exports = factory(require('angular'));
+    if (typeof module !== 'undefined' && module.exports) {
+    // CommonJS
+    if (typeof angular === 'undefined') {
```

Comment:
The idea here was that there would already be AMD shim configuration for Angular, so that the one and only instance would be available through `require('angular')`.